assignments/assignment7/faulty-oops.md

```diff
@@ -0,0 +1,43 @@
+Unable to handle kernel NULL pointer dereference at virtual address 0000000000000000
```

Comment:
@vaishnavijadhav19 use markdown to format this file, see https://github.com/cu-ecen-aeld/assignments-3-and-later-vaishnavijadhav19/blob/a04f17c6413ae90baa48ca1bfe7dca4d86600499/assignments/assignment7/faulty-oops.md for a view of how it renders today.